### PR TITLE
MISSIONS: integrate thruster autodeploy / launcher into launch & missions

### LINKED ISSUE #385: Write thruster auto deploy code
* [ ] Write a script/mission to deploy and retract thrusters (using the actuator board driver uf-mil/mil_common#133)
* [ ] Add the driver to hardware.launch
* [ ] Integrate into xbox controller

### PR DESCRIPTION
**Pull request by ironmig**
depends on uf-mil/mil_common#135
closes #385 

The follow changes are added to support remote thruster deploy/retraction
* Launch the actuator driver with the configuration for the thrusters
* Write missions to actuate the valves to allow for thruster deploy/retraction
* Connect the LB and RB buttons on xbox controller to retract and deploy thrusters, respectively

### CODE REVIEW

**Comment by ironmig** [comment]
Added missions for fireing / reloading launcher

File: utils/remote_control/navigator_keyboard_control/remote_control_lib/remote_control_lib.py [hunk 1]
```diff
@@ -120,6 +122,22 @@ def station_hold(self, *args, **kwargs):
...
+        def cb(terminal_state, result):
+            # TODO(ironmig): check that it was actually succesful
+            rospy.loginfo('Thrusters deployed!')
+
+        self.task_client.run_task('DeployThrusters', done_cb=cb)
+
+    @_timeout_check
+    def retract_thrusters(self, *args, **kwargs):
+        def cb(terminal_state, result):
+            # TODO(ironmig): check that it was actually succesful
```
Comment (Grymestone) [inline utils/remote_control/navigator_keyboard_control/remote_control_lib/remote_control_lib.py]: Was this a TODO before merge or after? Or is this a reminder to do a visual check of the thrusters after deploy/retract?

Comment (ironmig) [reply utils/remote_control/navigator_keyboard_control/remote_control_lib/remote_control_lib.py]: Good catch, added the check

File: mission_control/navigator_missions/navigator_missions/navigator.py [hunk 2]
```diff
@@ -57,6 +58,21 @@ class Navigator(BaseTask):
     green = "GREEN"
     blue = "BLUE"
 
+    # Time to wait for lock to engage when deploying thruster
```
[the remark this answers is not in the document]
Comment (ironmig) [reply mission_control/navigator_missions/navigator_missions/navigator.py]: Good idea, moved to ROS params

**Comment by ironmig** [comment]
Resolved merged conflicts and addressed comments

**Pull request merged**